LocalSettings.php

```diff
@@ -1507,6 +1507,72 @@
 			'mode' => 'packed',
 		],
 	],
+	
+	// GeoData
+	'wgMaxGeoSearchRadius' => [
+		'default' => 10000,
+		'gratispaideiawiki' => 20000,
+		'gratisdatawiki' => 20000,
+	],
+	'wgMaxCoordinatesPerPage' => [
+		'default' => 500,
+		'gratispaideiawiki' => 2000,
+		'gratisdatawiki' => 2000,
+	],
+	'wgDefaultDim' => [
+		'default' => 1000,
+		'gratispaideiawiki' => 2000,
+		'gratidatawiki' => 2000,
+	],
+	// Controls what GeoData should do when it encounters some problem
+	// Reaction type:
+	// 'track' - Add tracking category
+	// 'fail' - Consider the tag invalid, display message and add tracking category
+	// 'none' - Do nothing
+	'wgGeoDataWarningLevel' => [
```

Comment:
Why are we only.setting default? If it isn't custom, then this shouldn't be added, and left at extension default.

Reply:
@Universal-Omega this should be ready now